src/serac/physics/utilities/functional/tests/functional_comparison_L2.cpp

```diff
@@ -44,7 +47,7 @@ void functional_test(mfem::ParMesh& mesh, L2<p> test, L2<p> trial, Dimension<dim
   A.AddDomainIntegrator(new mfem::MassIntegrator(a_coef));
 
   // Assemble the bilinear form into a matrix
-  A.Assemble(0);
+  SERAC_PROFILE_VOID_EXPR(concat("mfem_localAssemble").c_str(), A.Assemble(0));
```

Comment:
does this `concat` do anything?

Reply:
Good catch. I forgot to add the `postfix` here.